src/main/java/com/artipie/http/MainSlice.java

```diff
@@ -112,30 +114,29 @@ public MainSlice(
     }
 
     /**
-     * Metrics storage Yaml node.
+     * Checks that metrics are collected by Prometheus.
+     *
      * @param settings Artipie settings
-     * @return Yaml node, could be null
+     * @return True if metrics are collected by Prometheus.
      */
-    private static Optional<YamlMapping> metricsStorage(final Settings settings) {
-        return Optional.ofNullable(settings.meta().yamlMapping("metrics"))
-            .flatMap(metrics -> Optional.ofNullable(metrics.yamlMapping("storage")));
+    static boolean isPrometheusConfigAvailable(final Settings settings) {
+        return settings
+            .meta()
+            .yamlSequence("metrics")
```

Comment:
if `metrics` section is absent at all, `yamlSequence("metrics")` returns null, and we will get NPE here